utils/devel/extract-perl

```diff
@@ -44,6 +44,7 @@ sub add_entry {
 }
 
 sub slurp{
+    no warnings 'once';
```

Comment:
Isn't there anything we can do about the file in fact being used once, instead of suppressing?

Reply:
Rewritten to use modern Perl and remove the no warning.